bigframes/ml/model_selection.py

```diff
@@ -147,3 +149,37 @@ def split(
             yield utils.convert_to_types(
                 [X_train, X_test, y_train, y_test], [X, X, y, y]
             )
+
+
+def cross_validate(
+    estimator,
+    X: Union[bpd.DataFrame, bpd.Series],
+    y: Union[bpd.DataFrame, bpd.Series, None] = None,
+    *,
+    cv: Optional[Union[int, KFold]] = None,
+) -> dict[str, list]:
+    if cv is None:
+        cv = KFold(n_splits=5)
+    elif isinstance(cv, int):
+        cv = KFold(n_splits=cv)
+
+    result: dict[str, list] = {"test_score": [], "fit_time": [], "score_time": []}
+    for X_train, X_test, y_train, y_test in cv.split(X, y):  # type: ignore
+        fit_start_time = time.time()
+        estimator.fit(X_train, y_train)
+        fit_time = time.time() - fit_start_time
```

Comment:
Maybe use perf_counter

From https://superfastpython.com/time-time-vs-time-perf_counter/:

The time.time() function should no longer be used for benchmarking because it is adjustable, non-monotonic, and lower resolution.